Trustroots/trustroots
# Support form: convert to React

Convert support form into React. Moves Angular->React migration forward and makes it possible to get form translated.

![image](https://user-images.githubusercontent.com/87168/103444857-c383a000-4c75-11eb-9d02-f2b194a8eea5.png)


#### Proposed Changes

* Convert Support form to React

#### Testing Instructions

* Open `/contact`
* Open `/support`
* Open above as logged in and non-logged in user
* Try appending `?report=somethin` to URLs
* When sent, confirm database entry in `supportrequests` MongoDB collection
* You can test error handling by changing from api.js file URL to something else (and cause 404 error)
* Confirm refreshing the page in a middle of message writing doesn't wipe out the message; it's persistent using a React hook that stores it in local storage
